Config.lua

```diff
@@ -307,18 +310,53 @@ local spells = {
         return not profile.general.area[info[2]].enabled
     end,
     args = {
+        selectedZone = {
+            name = L["Copy Settings From:"],
+            desc = L["Select the zone you want to copy settings from"],
+            order = 1,
+            type = "select",
+            values = function(info)
+                local tmpzones = {}
```

Comment:
I think we should add an "empty" value here, so it is possible to go back to not having the "Copy" button clickable again after having selected something and choosing to not copy anything.

Reply:
+1